### PR TITLE
Address logging

### LINKED ISSUE #72: Review logging and logging levels
#### Description

if there is an error use `.error` log if there otherwise use `.info` or `.debug` for additional information

#### DoD (Definition of Done)

review logging for existing source code.

#### Testing

Peer review

### PR DESCRIPTION
**Pull request by blankdots**
### Related issues

Fixes #72 

### Type of change

- [x] New feature (non-breaking change which adds functionality)

### Changes Made

Added error, info and debug logs to functions

### Testing

- [x] Tests do not apply

### Mentions

debug log might be verbose but it is good for tracing if functions were done.

### CODE REVIEW

File: metadata_backend/api/operators.py [hunk 1]
```diff
@@ -230,6 +243,7 @@ async def _format_data_to_create_and_add_to_db(self, schema_type: str,
         data["dateModified"] = datetime.utcnow()
         if schema_type == "study":
             data["publishDate"] = datetime.utcnow() + relativedelta(months=2)
+        LOG.debug(f"Operator format data for {schema_type} to add to DB")
```
Comment (otahontas) [inline metadata_backend/api/operators.py:246]: format -> formatted

File: metadata_backend/api/operators.py [hunk 2]
```diff
@@ -315,6 +330,7 @@ async def _format_data_to_create_and_add_to_db(self, schema_type: str,
         accession_id = (await Operator(db_client).
                         _format_data_to_create_and_add_to_db(schema_type,
                                                              data_as_json))
+        LOG.debug(f"XMLOperator format data for {schema_type} to add to DB")
```
Comment (otahontas) [inline metadata_backend/api/operators.py:333]: format -> formatted

File: metadata_backend/database/db_service.py [hunk 3]
```diff
@@ -142,6 +147,7 @@ def query(self, collection: str, query: Dict) -> AsyncIOMotorCursor:
         :param query: query to be used
         :returns: Async cursor instance which should be awaited when iterating
         """
+        LOG.debug("DB doc query performed.")
```
Comment (otahontas) [inline metadata_backend/database/db_service.py:150]: Query doesn't actually happen here, it is done when data is returned from function here: https://github.com/CSCfi/metadata-submitter/blob/ea72ccddf7c9cfbd1aa2833adf773a7e6e8ba8d8/metadata_backend/api/operators.py#L213

Maybe move logging there?

File: metadata_backend/api/operators.py (hunk 1 above)
Comment (blankdots) [reply metadata_backend/api/operators.py]: done.

File: metadata_backend/api/operators.py (hunk 2 above)
Comment (blankdots) [reply metadata_backend/api/operators.py]: done.

File: metadata_backend/database/db_service.py (hunk 3 above)
Comment (blankdots) [reply metadata_backend/database/db_service.py:150]: it was meant to signal that a query was called, but will add logging there as suggested.

Comment (blankdots) [reply metadata_backend/database/db_service.py:150]: a bit confused on what to add there because there is already logs ... :confused:

Comment (otahontas) [reply metadata_backend/database/db_service.py:150]: you're right, my bad. This is ok now

**Pull request merged**